types/bull-arena/index.d.ts

```diff
@@ -1,56 +1,80 @@
-// Type definitions for bull-arena 2.6
+// Type definitions for bull-arena 3.0
 // Project: https://github.com/bee-queue/arena/
 // Definitions by: Levi Bostian <https://github.com/levibostian>
+//                 Eli Skeggs <https://github.com/skeggse>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
```

Comment:
Is this just the lowest version supported by these types?

Reply:
Yes, and the lowest value on Definitely Typed is actually 3.0, which will increase to 3.1 this week for the release of Typescript 4.0. Since this package is intended to work on 2.8 and up, you can remove this line if you want.